examples/docking-protein-ligand/docking-protein-ligand.cfg

```diff
@@ -26,27 +26,30 @@ delenph = false
 
 [rigidbody]
 ambig_fname = './data/ambig-active-rigidbody.tbl'
-sampling = 20
+ligand_param_fname = "./data/ligand-prodrg.param"
+ligand_top_fname = "./data/ligand-prodrg.top"
+sampling = 5
 noecv = true
-w_vdw = 1.0 
+w_vdw = 1.0
 
-[caprieval]
-reference = './data/target.pdb'
+# (28/11/21) no support for protein-ligand yet
+# [caprieval]
+# reference = './data/target.pdb'
 
 [seletop]
 select = 5
 
 [flexref]
 ambig_fname = './data/ambig-passive.tbl'
+ligand_param_fname = "./data/ligand-prodrg.param"
+ligand_top_fname = "./data/ligand-prodrg.top"
```

Comment:
There's no need to add the `./` before the path.

Reply:
old habits die hard 😅